src/main/java/de/tum/in/www1/artemis/repository/UserRepository.java

```diff
@@ -121,18 +116,23 @@ OR lower(user.login) = lower(:#{#searchInput}))
     Set<User> findAllInGroup(@Param("groupName") String groupName);
 
     @Query("""
-            SELECT DISTINCT user
+            SELECT NEW de.tum.in.www1.artemis.domain.ConversationWebSocketRecipientSummary (
+                user,
+                CASE WHEN cp.isHidden = true THEN true ELSE false END,
```

Comment:
Is the case select needed here?
```suggestion
                cp.isHidden
```